### PR TITLE
[Rule Tuning] Account Password Reset Remotely

### PR DESCRIPTION
**Pull request by Samirbous**
https://github.com/elastic/detection-rules/issues/3332

- reduced maxspan from 5m to 1m (automated pwd reset)
- excluded most observed noisy winlog.event_data.TargetUserName patterns (IAM/AD service account dedicated for mass pwd reset)
- added a note for performance impact

### CODE REVIEW

**Review by terrancedejesus — APPROVED** [review]

File: rules/windows/persistence_remote_password_reset.toml [hunk 1]
```diff
@@ -14,10 +14,11 @@ passwords to maintain access or evade password duration policies and preserve co
 """
 false_positives = ["Legitimate remote account administration."]
 from = "now-9m"
-index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+index = ["winlogbeat-*", "logs-system.security*", "logs-windows.forwarded*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Account Password Reset Remotely"
+note = "This rule may cause medium to high performance impact due to logic scoping all remote Windows logon activity."
```
Comment (brokensound77) [inline rules/windows/persistence_remote_password_reset.toml:21]: ++

Comment (brokensound77) [reply rules/windows/persistence_remote_password_reset.toml:21]: I wonder if this is better in guide or setup ...

Thoughts @terrancedejesus @w0rk3r

**Review by brokensound77 — APPROVED** [review]
One question, otherwise LGTM

File: rules/windows/persistence_remote_password_reset.toml (hunk 1 above)
Comment (terrancedejesus) [reply rules/windows/persistence_remote_password_reset.toml:21]: I think it's best where it is...a note about performance.

**Pull request merged**